cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import argparse
+import subprocess
+import sys
+import os.path
+import shlex
+
+def _run_command(cmd):
+    try:
+        res = subprocess.run(shlex.split(cmd), shell=False)
+        if res.returncode:
+            print(f"failed to run command {cmd} ")
+            sys.exit(1)
```

Comment:
`subprocess.run` can handle exit non-zero with and exception.
Use parameter `check=True`
https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
Addressed